tests/NHSD.BuyingCatalogue.Identity.Api.IntegrationTests/Utils/Database.cs

```diff
@@ -11,17 +12,31 @@ namespace NHSD.BuyingCatalogue.Identity.Api.IntegrationTests.Utils
     public sealed class Database
     {
         [BeforeScenario]
-        public async Task Reset(IConfigurationRoot config)
+        public async Task Reset(IConfigurationRoot config, ScenarioContext context)
         {
             using IDbConnection databaseConnection = new SqlConnection(config.GetConnectionString("CatalogueUsersAdmin"));
             await databaseConnection.ExecuteAsync("ALTER ROLE db_owner ADD MEMBER [NHSD];");
             await databaseConnection.ExecuteAsync("DELETE FROM Organisations");
+            await databaseConnection.ExecuteAsync("DELETE FROM AspNetUsers WHERE OrganisationFunction = 'TestUser'");
```

Comment:
Parallel running is for sure affected currently. I can add a task to look at that, it's not going to be super simple, just because if multiple features have a user with the same email, we'll get key restraints kicking in. I have a few ideas that could work... I'll create a task!